pytorch_toolkit/ote/readme.md

```diff
@@ -193,5 +196,39 @@ optional arguments:
                         Additional args to OpenVINO Model Optimizer
 ```
 
-## quantize.py
-TBD
+## compress.py
+Note that command line parameters of `compress.py` similar to command line parameters
+of `train.py`, but without parameters `--epochs` and `--base-learning-rate`,
+since these parameters for compression are set internally.
```

Comment:
@Ilya-Krylov just one more reason not to expose these parameters to the template.